### PR TITLE
IGAPP-1250: Campaign logo mobile

### PR DESCRIPTION
**Pull request by f1sh1918**
Switch the campaign start date to test if its working

I adjusted the symbol link to use `integreat/asset` folder for `integreat-test-cms` and `integreat-e2e` to automatically update the assets by adding to `integreat/assets`

### CODE REVIEW

File: build-configs/integreat/index.ts [hunk 1]
```diff
@@ -103,8 +103,9 @@ export const webIntegreatBuildConfig: WebBuildConfigType = {
   },
   campaign: {
     campaignAppLogo: '/campaign-app-logo.png',
-    startDate: '2021-03-08T00:00:00.000Z',
-    endDate: '2021-03-15T00:00:00.000Z',
+    campaignAppLogoMobile: '/campaign-app-logo-mobile.png',
+    startDate: '2023-03-08T00:00:00.000Z',
```
Comment (f1sh1918) [inline build-configs/integreat/index.ts:107]: For testing purpose change to f.e.
`'2023-02-08T00:00:00.000Z'`

File: build-configs/integreat-test-cms/assets/campaign-app-logo-mobile.png [hunk 2]
```diff
@@ -0,0 +1 @@
+/Users/afischer/projects/integreat-app/build-configs/integreat/assets/campaign-app-logo-mobile.png
```
Comment (SteffiStuffel) [inline build-configs/integreat-test-cms/assets/campaign-app-logo-mobile.png:1]: please fix.

**Review by sarahsporck — COMMENTED** [review]
The logo is not displayed in native. I guess you still need to change it in the header

**Comment by f1sh1918** [comment]
> The logo is not displayed in native. I guess you still need to change it in the header
> 
> EDIT: Just realized that you added the mobile image to the web mobile version. Any reason why we don't display the logo in native?

Simply it was never a requirement to do so. It was never implemented on native.I asked Sascha and i didn't get any additional requirement

File: build-configs/integreat-test-cms/assets/campaign-app-logo-mobile.png (hunk 2 above)
Comment (f1sh1918) [reply build-configs/integreat-test-cms/assets/campaign-app-logo-mobile.png]: i adjusted the symlink to use `integreat/assets` folder so if we add new assets the gonna be symlinked auto

File: web/src/components/HeaderLogo.tsx [hunk 3]
```diff
@@ -52,8 +52,7 @@ export const HeaderLogo = ({ link }: HeaderLogoProps): ReactElement => {
   const currentDate = moment()
   const showCampaignLogo = campaign && currentDate.isAfter(campaign.startDate) && currentDate.isBefore(campaign.endDate)
   const src = campaign && showCampaignLogo ? campaign.campaignAppLogo : icons.appLogo
-  const srcMobile = campaign && showCampaignLogo ? campaign.campaignAppLogo : icons.appLogoMobile
-  // TODO delete campaign in one commit incl assets
+  const srcMobile = campaign && showCampaignLogo ? campaign.campaignAppLogoMobile : icons.appLogoMobile
```
Comment (SteffiStuffel) [inline web/src/components/HeaderLogo.tsx:55]: Please remove the `campaign &&` check, as this is already checked inside `showCampaignLogo`.

**Review by SteffiStuffel — APPROVED** [review]
Looks good in Chrome, desktop and mobile. Also checked if web-e2e will work during the campain.

**Review by sarahsporck — APPROVED** [review]
I guess not having native support is fine by me. But it would be nice to implement campaigns on both native and mobile.
Maybe we kept the symlinks before to enable developing on windows computers natively. With this change you can now only develop on Windows with wsl, I think.

**Pull request merged**